app/assets/stylesheets/provider/_flash.scss

```diff
@@ -1,8 +1,12 @@
 #flashWrapper {
-  position: absolute;
-  right: line-height-times(1);
-  text-align: right;
-  @include animation(fadeInLeft 0.5s ease-in-out);
+  @include animation(fadeInRight 0.5s ease-in-out);
```

Comment:
@HemaHG tests are failing because you changed `fadeInLeft` to `fadeInRight` here, but not in line 45 of this file
https://github.com/3scale/porta/blob/87c5e3253eec562d717e89adc32ff81939ce188b/app/assets/stylesheets/provider/_flash.scss#L45